src/main/java/kubatech/mixin/mixins/minecraft/LocaleMixin.java

```diff
@@ -0,0 +1,47 @@
+package kubatech.mixin.mixins.minecraft;
+
+import static kubatech.mixin.MixinsVariablesHelper.currentlyTranslating;
+
+import java.util.regex.Matcher;
+
+import net.minecraft.client.resources.Locale;
+import net.minecraft.util.ResourceLocation;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import kubatech.Tags;
+
+@SuppressWarnings("unused")
+@Mixin(value = Locale.class)
+public class LocaleMixin {
+
+    @ModifyArg(
+        method = "Lnet/minecraft/client/resources/Locale;loadLocaleDataFiles(Lnet/minecraft/client/resources/IResourceManager;Ljava/util/List;)V",
+        at = @At(
+            value = "INVOKE",
+            target = "Lnet/minecraft/client/resources/IResourceManager;getAllResources(Lnet/minecraft/util/ResourceLocation;)Ljava/util/List;"),
+        index = 0,
+        require = 1)
+    private ResourceLocation loadLocaleDataFiles(ResourceLocation resourceLocation) {
+        currentlyTranslating = resourceLocation.getResourceDomain();
+        return resourceLocation;
+    }
+
+    @Redirect(
+        method = "Lnet/minecraft/client/resources/Locale;loadLocaleData(Ljava/io/InputStream;)V",
```

Comment:
`Unnecessary qualified reference to 'loadLocaleData' in target class`
```suggestion
        method = "loadLocaleData(Ljava/io/InputStream;)V",
```

Reply:
Done